src/api/Controllers/UploadController.cs

```diff
@@ -232,6 +286,66 @@ private void ValidateBoreholeImports(int workgroupId, List<BoreholeImport> boreh
         }
     }
 
+    private void ValidateLithologyImports(List<int> importIds, List<LithologyImport>? lithologyImports = null)
+    {
+        if (lithologyImports == null || lithologyImports.Count == 0) return;
+
+        var nullOrEmptyMsg = "Field '{0}' is required.";
```

Comment:
REQ: In eine `readonly` Variable auslagern und in `ValidateBoreholeImports` wiederverwenden.